contracts/libraries/VaultLifecycle.sol

```diff
@@ -411,16 +411,15 @@ library VaultLifecycle {
     function verifyConstructorParams(
         address owner,
         address feeRecipient,
-        uint256 performanceFee,
         uint256 managementFee,
```

Comment:
Since we're not checking `managementFee > 0` we can just remove it entirely right?